test/foundry/modules/royalty/RoyaltyPolicyLAP.t.sol

```diff
@@ -146,7 +147,12 @@ contract TestRoyaltyPolicyLAP is BaseTest {
     }
 
     function test_RoyaltyPolicyLAP_setSnapshotInterval_revert_NotOwner() public {
-        vm.expectRevert(Errors.Governance__OnlyProtocolAdmin.selector);
+        vm.expectRevert(
+            abi.encodeWithSelector(
+                IAccessManaged.AccessManagedUnauthorized.selector,
+                address(royaltyModule) // How is this the caller??
```

Comment:
Not sure why this is the case, cc @Spablob. Could this be an issue with variable value assignment somewhere?

Reply:
This is just because on line 25, it's pranking the royalty module address to be the caller.

This is because multiple functions in RoyaltyPolicyLAP require royaltyModule to be the caller so it was pranked in setup

Reply:
I changed the test stopping the prank and adding it before it's needed, to save on debug time for future devs